04-loop.md

```diff
@@ -230,6 +231,16 @@ the `head` and `tail` combination selects lines 81-100 from whatever file is bei
 > ~~~
 >
 > but it's simpler just to avoid using spaces (or other special characters) in filenames.
+> 
+> If you have a series of files with spaces, you can run a loop to rename files:
+~~~ {.bash}
+for filename in *
+do 
+	mv "$filename" "${filename/ /_}"
+done
+~~~
+> this will scan all the files in the directory for white spaces and replace them with underscore.
```

Comment:
Without having real files to try this on, this command wouldn't work. Also it's quite cryptic (no explanation about the `/ /_`).